sonos.php

```diff
@@ -3,414 +3,398 @@
 
 class Sonos
 {
-	protected $_config;
-	protected $_PHPSonos;
-        protected $_zone_ip;
-
-	function __construct()
-	{
-		$this->_getConfig();
-		$this->_main();
-	}
-	
-	protected function _getConfig()
-	{
-		if (file_exists('config.php')) 
-		{
-			include 'config.php';
-			$this->_config = $config;
-		}
-		else
-		{
-			throw new Exception('Config File [config.php] does not exists.');
-		}		
-	}
-	
-	protected function _getPHPSonos()
-	{
-		if (file_exists($this->_config['filePhpSonos'])) 
-		{
-			include $this->_config['filePhpSonos'];
-		}
-		else
-		{
-			throw new Exception('File [PHPSonos.inc.php] does not exists.');
-		}		
-	}
-	
-	protected function _main()
-	{
-                if($this->_config['logging'])
-                {
-                        $this->_log();
-                }
-                
-                //for compability with old script
-                if(!empty($_GET['do']))
-                {
-                        $action = $_GET['do'];
-                }
-                else
-                {
-                        $action = !empty($_GET['action']) ? $_GET['action'] : '';
-                }
-                
-		$zone = !empty($_GET['zone']) ? $_GET['zone'] : '';
-		
-		$this->_zone_ip = $this->_assertZone($zone);
-		
-		$this->_getPHPSonos();
-		$this->_PHPSonos = new PHPSonos($this->_zone_ip);		
-		
-		$this->_assertAction($action);
+    protected $_config;

```

Comment:
Thanks for your Pull request. Nice feature added. Is it possible to incorporate your changes without changing the formatting of the whole class?